pkg/netpol/connlist/connlist.go

```diff
@@ -260,24 +304,45 @@ func (c *connection) ProtocolsAndPorts() map[v1.Protocol][]eval.PortRange {
 	return c.protocolsAndPorts
 }
 
-// return a string representation for a connection object
+// return a txt representation for a connection object
+func (c *connection) TxtFormat() string {
+	connStr := c.String()
```

Comment:
Is there a reason not to use `c.String()` as the text format?

Reply:
actually `String()` only converted the protocols and ports fields of the connection to string ,replaced by `getProtocolsAndPortsStr()`